pkgs/development/libraries/raft/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkgconfig
+, libuv
+}:
+
+let
+  pname = "raft";
+  version = "0.9.5";
```

Comment:
i would move the pname and version into the mkDerivation scope, and make it recursive (rec)

Reply:
I prefer not to use `rec`, it sacrifices overrides for nothing but syntactical convenience.

Reply:
? you can still use overrideAttrs just fine